tutorials/algorithms/09_textbook_algorithms.ipynb

```diff
@@ -4,13 +4,11 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# Textbook and Shor's algorithms\n",
+    "# Shor's algorithms\n",
     "\n",
-    "Qiskit contains implementations of the well known textbook quantum algorithms such as the [Deutsch-Jozsa algorithm](https://qiskit.org/textbook/ch-algorithms/deutsch-jozsa.html), the [Bernstein-Vazirani algorithm](https://qiskit.org/textbook/ch-algorithms/bernstein-vazirani.html) and [Simon's algorithm](https://qiskit.org/textbook/ch-algorithms/simon.html).\n",
+    "Qiski has an implementation of [Shor's algorithm](https://qiskit.org/textbook/ch-algorithms/shor.html).\n",
```

Comment:
Should we rename the notebook itself too - say `09_factorizers` since this is all this notebook is about now